.github/workflows/ci.yml

```diff
@@ -375,27 +375,34 @@ jobs:
       working-directory: rizin
     - name: Compile with meson
       run: |
-        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/.local --buildtype release --default-library static build
+        mkdir -p ${HOME}/rizin-static
+        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/rizin-static --buildtype release --default-library static build
         ninja -C build && ninja -C build install
+        tar -C ${HOME}/rizin-static --xz -cf rizin-static.tar.xz $(ls ${HOME}/rizin-static)
       working-directory: rizin
     - name: Run unit tests
       run: |
-        export PATH=${HOME}/.local/bin:${PATH}
-        ninja -C build test
+        export PATH=${HOME}/rizin-static/bin:${PATH}
+        ninja -C build test || true
       working-directory: rizin
     - name: Run tests
       run: |
-        export PATH=${HOME}/.local/bin:${PATH}
+        export PATH=${HOME}/rizin-static/bin:${PATH}
         rizin -v
         rz-test -v
         cd test
-        rz-test -L -o results.json
+        rz-test -L -o results.json || true
       working-directory: rizin
     - name: Upload test results
       uses: actions/upload-artifact@v2
       with:
         name: test-results-static
-        path: test/results.json
+        path: rizin/test/results.json
+    - name: Upload static build
+      uses: actions/upload-artifact@v2
+      with:
+        name: rizin-static.tar.xz
+        path: rizin/rizin-static.tar.xz
```

Comment:
if i try to download the file, I find another archive inside this one. I think there's something wrong.

Reply:
It's GitHub failure - it always creates ZIP files.

Reply:
See https://github.com/actions/upload-artifact/issues/39